### PR TITLE
vkcube: Add more debug utils object names and cb labels

### PR DESCRIPTION
**Pull request by jeremyg-lunarg**

### CODE REVIEW

File: cube/cube.c [hunk 1]
```diff
@@ -2002,6 +2034,7 @@ static VkShaderModule demo_prepare_shader_module(struct demo *demo, const uint32
 
     err = vkCreateShaderModule(demo->device, &moduleCreateInfo, NULL, &module);
     assert(!err);
+    demo_name_object(demo, VK_OBJECT_TYPE_SHADER_MODULE, (uint64_t)module, "%s", name);
```
Comment (charles-lunarg) [inline cube/cube.c:2037]: Couldn't this be simplified to pass name as the string directly? For example:
```c
    demo_name_object(demo, VK_OBJECT_TYPE_SHADER_MODULE, (uint64_t)module, name);
```

**Review by charles-lunarg — COMMENTED** [review]
Just a general comment, but its a good idea to check the error code returned from `SetDebugUtilsObjectNameEXT`, as well as make sure any other functions that return VkResult are being checked. I found `vkGetPhysicalDeviceSurfaceSupportKHR` (line 3714) isn't being checked, and I'm sure there are others. Not a requirement, just a thought.

File: cube/cube.c (hunk 1 above)
Comment (jeremyg-lunarg) [reply cube/cube.c:2040]: Some code checking tools become angry when you pass a non-literal string as the format argument to a printf-like function.

Comment (charles-lunarg) [reply cube/cube.c:2040]: Ahhh okay, I figured that might of been the case. Good to know for sure.

**Pull request merged**